lunes_cms/cms/migrations/0005_alter_document_word_type.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.13 on 2022-06-29 10:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cms", "0004_feedback"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="document",
+            name="word_type",
+            field=models.IntegerField(
+                choices=[(0, "Nomen"), (1, "Verb"), (2, "Adjektiv")],
+                default="",
+                verbose_name="word type",
+            ),
+        ),
+    ]
+
+    document_dict = {0: "Nomen", 1: "Verb", 2: "Adjektiv"}
+
+    document_bulk_update_list = []
+
+    for key, value in document_dict.items():
+        document = models.Document.objects.get(id=key)
+        document.word_type = value
+
+        # append the updated user object to the list
+        document_bulk_update_list.append(document)
```

Comment:
You cannot put the python code directly into the class definition - this way, the code will run immediately after a class has been imported.
Instead, you need to outsource the code into a function and then reference this function inside the migration class as described here: https://docs.djangoproject.com/en/3.2/ref/migration-operations/#runpython